template/package.json

```diff
@@ -12,15 +12,18 @@
     "pretty": "prettier --write \"./**/*.{js,jsx,json}\"",
     "prepare": "husky install",
     "postinstall": "bash ./scripts/post-install.sh",
-    "reverse": "adb reverse tcp:8081 tcp:8081"
+    "reverse": "adb reverse tcp:8081 tcp:8081",
+    "plop": "plop"
   },
   "dependencies": {
     "@react-native-async-storage/async-storage": "^1.17.10",
     "@react-navigation/native": "^6.0.13",
     "@react-navigation/native-stack": "^6.9.1",
     "@reduxjs/toolkit": "^1.8.6",
     "i18n-js": "^4.1.1",
+    "i18next": "^22.4.9",
     "react": "18.1.0",
+    "react-i18next": "^12.1.4",
```

Comment:
Should remove react-i18next, because this is generate file feature

Reply:
updated